android/gradle.properties

```diff
@@ -10,8 +10,7 @@
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
 # Default value: -Xmx1024m -XX:MaxPermSize=256m
-# org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
-org.gradle.jvmargs=-Xmx512m -XX:+HeapDumpOnOutOfMemoryError
+org.gradle.jvmargs=-Xmx1024m -XX:MaxPermSize=256m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
```

Comment:
Do we need to keep these arguments in here, or are they actually "defaults"? (I.e. would removing them mean the same behavior?)

Reply:
Well, the "default" line would be doubled RAM and wouldn't dump on OOM (I recall adding the dump-on-OOM because that was actually an issue where it would OOM and not tell me.)

So I don't know? But I don't think I want the defaults.